cypress/integration/gpu-dependant/canvas.js

```diff
@@ -0,0 +1,80 @@
+// Smoke test for Tectonic Explorer
+
+context('Canvas Test', () => {
+  before(function () {
+    cy.visit('/?planetWizard=true')
+    cy.wait(3000)
+  })
+
+  context('Set up same as smoke test to draw a small continent', () => {
+    it('Draws a small land mass on the planet', () => {
+      cy.get('.planet-wizard-overlay > :nth-child(1)').click()
+      // cy.get('.interaction-selector > :nth-child(3)').click()
+      cy.wait(7000)
+      cy.get('canvas').eq(1)
+        .trigger('mousedown', { force: true }, { which: 1, pageX: 500, pageY: 300 })
+        .trigger('mousemove', { force: true }, { which: 1, pageX: 350, pageY: 350 })
+        .trigger('mouseup', { force: true })
```

Comment:
This dragging code doesn't really work. It somehow randomly draws something in mouse down position.
I've also added a helper with new implementation: `cy.mainCanvasDrag([<list of mouse positions>])`.